test/src/org/labkey/test/tests/cds/CDSVisualizationPlotTest.java

```diff
@@ -650,7 +650,7 @@ public void verifyPKLinePlot() throws Exception
         coloraxis.confirmSelection();
 
         plotGreenLineCount = getElementCount(greenLineLoc);
-        assertEquals("Line should also use color for a non-Subject Characteristic field that won't change over time", 27, plotGreenLineCount);
+        assertEquals("Line should also use color for a non-Subject Characteristic field that won't change over time", 7, plotGreenLineCount);
```

Comment:
Did something change in the product? How come the expected number reduced?

Reply:
The test data changed with the addition of the LLOQ fields. Overall the plot is correct : 34 subjects with 27 in one category and 7 in the other. It's just that with the new fields causing a shift in the category color there are 27 black and 7 green lines instead of 27 green and 7 black lines. Took me a while to figure it out until Cory and Xing helped me.